packages/app/src/app/pages/NewDashboard/Content/routes/Settings/TeamSettings.tsx

```diff
@@ -370,8 +431,8 @@ export const TeamSettings = () => {
                         userId: user.id,
                       }),
                   },
-                ].filter(Boolean);
-              }}
+                ].filter(Boolean)
```

Comment:
No need for the extra filtering, since all elements are objects

```suggestion
                ]
```